src/Runner.java

```diff
@@ -0,0 +1,68 @@
+import java.util.Comparator;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Arrays;
+
+public class Runner {
+   public static void main(String[] args) {
+      try(Scanner sc = new Scanner(new File("src/in.txt"))) {
+
+
+         Purchase[] purchases;
+         final int PURCHASES_NUMBER = sc.nextInt();
+         purchases = new Purchase[PURCHASES_NUMBER];
+         //инициализация массива из файла в цикле со счетчиком
+	     for (int i = 0; i < PURCHASES_NUMBER; i++){
+            purchases[i] = new Purchase();
+            purchases[i].setAmount(sc.nextInt());
+            purchases[i].setDiscount(sc.nextInt());
+            purchases[i].setWeekDay(WeekDay.values()[sc.nextInt()]);
+         }
+         printPurchases(purchases);
+         int totalCost = 0;		//инициализация финансовых величин
+         int totalCostMonday = 0;
+         int maxCost = 0;
+         double meanCost = 0.0;	//среднее - это статистическая, а не финансовая величина!
+         WeekDay maxCostDay = null;
+         //обработка массива (пункт 4) в цикле for-each (без счетчика)
+         for (Purchase purchase : purchases){
+            totalCost += purchase.getCost();
+            if (purchase.getWeekDay() == WeekDay.MONDAY) totalCostMonday += purchase.getCost();

```

Comment:
please, use brackets after if
also, why you use totalCostMonday  second time when you are searching monday price?